plugins/flatpak/tests/build.py

```diff
@@ -70,6 +71,13 @@ def copy_repo(srcdir, destdir):
         shutil.rmtree(destdir_repo)
     shutil.copytree(srcdir_repo, destdir_repo)
 
+# use git master where available
+local_checkout = '/home/hughsie/Code/flatpak'
+if os.path.exists(local_checkout):
+    flatpak_cmd = os.path.join(local_checkout, 'flatpak')
+else:
+    flatpak_cmd = 'flatpak'
```

Comment:
I dont like the fact that `flatpak_cmd` is a global var used by `build_flatpak` and `build_flatpak_bundle`, but details.

Reply:
In the upstream PR there's a commit that deletes these few lines.